GitTfs.Setup/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Diagnostics;
+using WixSharp;
+using System.IO;
+using File = WixSharp.File;
+
+namespace GitTfs.Setup
+{
+    class Program
+    {
+        static void TxtToRtf(string inputFile, string outputFile) {
+            var lines = System.IO.File.ReadAllLines(inputFile);
+
+            using (var f = new StreamWriter(outputFile))
+            {
+                f.WriteLine(@"{\rtf1\ansi\deff0{\fonttbl{\f0\fnil\fcharset0 Arial;}}\nowwrap\fs18");
+                foreach (var l in lines)
+                {
+                    f.WriteLine(l);
+                    f.WriteLine(@"\line ");
+                }
+                f.WriteLine(@"}");
+            }
+        }
+
+        static void Main()
+        {
+#if DEBUG
+            var ProjectName = "GitTfsDebug";
+            var GUID = new Guid("6FE30B47-2577-43AD-9095-1861BA25889B");
+            var Conf = "Debug";
+#else
+            var ProjectName = "GitTfs";
+            var GUID = new Guid("98823CC2-0C2E-4CF7-B5ED-EA2DD26559BF");
+            var Conf = "Release";
+#endif
```

Comment:
I avoid preprocessor conditionals because they make it so that different code is compiled in debug and release. It makes it easier to accidentally add a bug that affects release without noticing the problem in debug.